index.html

```diff
@@ -703,15 +703,17 @@ <h2 class='informative'>Implementation Patterns</h2>
     </section>
 
     <section id="messaging-spi">
-      <h2>Messaging SPI</h2>
```

Comment:
As you mentioned, @zimeon, the Messaging section is different than the others in that it details what a server implementation must provide, not how a user "interfaces" with the capability. 
The inclusion of "SPI" tries to highlight that difference. 
Is your opinion that the difference in nature of this section from the others need not be emphasized?